src/main/java/net/mcreator/workspace/Workspace.java

```diff
@@ -294,6 +294,21 @@ public long getMCreatorVersion() {
 		fileManager.close();
 	}
 
+	@Override public boolean equals(Object o) {
+		if (this == o)
+			return true;
+
+		if (o instanceof Workspace other) {
+			return getWorkspaceFolder().equals(other.getWorkspaceFolder());
+		}
+
+		return false;
+	}
+
+	@Override public int hashCode() {
+		return getWorkspaceFolder().hashCode();

```

Comment:
This may also need some second thought if it is safe

Reply:
https://docs.oracle.com/javase/8/docs/api/java/io/File.html#hashCode-- is a bit confusing in wording

Reply:
![image](https://github.com/user-attachments/assets/162e8181-890e-4cc2-af29-181e8316a42d)

Ok hashcode is fine likely